src/components/Table/Table.tsx

```diff
@@ -0,0 +1,50 @@
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const Table: React.FC<Props> = ({ person }) => {
```

Comment:
It doesn't look like a table, it's only a row in a table with person info. Maybe you can call as PersonRow